### DISCUSSION TITLE
BlueSCSI not seen by Mac Plus

### DISCUSSION THREAD

**Discussion by sjoerdvanderhoorn** [BlueSCSI/BlueSCSI-v2#217] [Help] [answered]
_This is a split off from https://github.com/BlueSCSI/BlueSCSI-v2/discussions/156#discussioncomment-11357838_

When I try to boot my Macintosh Plus with the 25pin BlueSCSI v2, It will not get past the question mark-floppy. Actually, I have two Pluses (is that proper plural?) and neither of them boot from the BlueSCSI device even though both will boot from a regular bootdisk.

Content of `bluescsi.ini`:

```ini
[SCSI]
System=MacPlus
```

Images tried:
* Every image from [the MacPack](https://archive.org/details/macpack)
* HD20_512-1GB-MacPlus-7+7.hda.zip from https://mega.nz/folder/8hA3AQCJ#pWUq92L70yDXlogy9lk5Dg
* HD0-OpenRetroSCSI-6.0.8-500M.hda.zip from https://mega.nz/folder/8hA3AQCJ#pWUq92L70yDXlogy9lk5Dg
* https://68kmla.org/bb/index.php?threads/bluescsi-questions-for-mac-plus.41840/#post-482168
* https://www.savagetaylor.com/wp-content/uploads/68k_Macintosh/Bootdisks/608_2GB_drive.zip
* https://macintoshgarden.org/apps/piscsi-rascsi
* ... and various others

When attached to a power outlet via USB and plugged in to the Mac Plus, I am consistently only seeing the PWR led ON, but nothing on the activity LED. When I boot my systems with the boot disk, it will also not show a connected hard drive. I'm using a Western Digital SD Purple but tried two other unbranded SD cards I had around with similar results.

Below is a copy of the `log.txt` file from when I enabled `Debug=1` . It will properly read the disk image(s) after copying them onto the sdcard and show `Initialization complete!`. If I then boot up the Macintosh Plus it will get stuck in this eternal `DBG -- BUS_BUSY` --> `DBG -- BUS_FREE` loop.

```
Platform: BlueSCSI
FW Version: 2024.10.26-rel Oct 26 2024 22:02:38
Flash chip size: 2048 kB
 
=== SD Card Info ===
SD card detected, exFAT volume size: 30427 MB
SD Name: WX32G, MID: 0x03, OID: 0x53 0x44
[229ms]  
[230ms] === Global Config ===
[233ms] Reading configuration from bluescsi.ini
[237ms] Active configuration (using system preset "MacPlus"):
[242ms] -- EnableSCSI2 is off
[245ms] -- EnableSelLatch is on
[248ms] -- Debug is enabled
[250ms] DBG -- Parity is enabled
[253ms] DBG -- ReinsertCDOnInquiry is disabled
[369ms]  
[369ms] === Finding images in / ===
[372ms] == Opening /HD20_512 RaSCSI-Boot-6.0.8.hda for ID: 2 LUN: 0
[379ms] DBG ---- Read prefetch enabled: 8192 bytes
[382ms] DBG ---- Valid Macintosh Device Image detected.
[385ms] ---- Image ready
[407ms] 
[407ms] === ROM Drive ===
[409ms] Platform supports ROM drive up to 1692 kB
[414ms] ---- ROM drive image not detected
[417ms]  
[418ms] === Configured SCSI Devices ===
[422ms] * ID: 2, BlockSize: 512, Type: Fixed, Quirks: Apple, Size: 40960kB
[1221ms]  
[1221ms] Initialization complete!
[1239ms] INFO: Pico Voltage: 3.217V.
[10091ms] DBG BUS RESET
[10336ms] DBG -- BUS_BUSY
[10336ms] DBG -- BUS_FREE
[10338ms] DBG -- BUS_BUSY
[10340ms] DBG -- BUS_FREE
...
```

I went through the entire [troubleshooting](https://bluescsi.com/docs/Troubleshooting) page except for creating my own image using Disk Jockey. 

I confirmed that the BlueSCSI is properly plugged in. I can not yet check for corrossion and have ordered tools to open up my devices, but please note that I own **two** MacPlusses and they both have the same issue. I am leaning towards the BlueSCSI itself being the issue but I have no easy way to verify that. Also, since the log does display BUS_FREE / BUS_BUSY entries around the time I boot up the machine there must at least be some communication going on.

I also tested with debug off and the [last nightly build](https://github.com/BlueSCSI/BlueSCSI-v2/releases/tag/latest).

Any chance someone spots something I've missed, or can tell if there is a disk image I should have tried instead?

**Comment by erichelgeson** [comment]
> Any chance someone spots something I've missed, or can tell if there is a disk image I should have tried instead?

Trying more disk images wont change anything - the first 3 work with a Mac Plus. 

Other thing I could think of to try: 
If you had any other SCSI device to connect it and see if that is seen.
Make sure your USB power is 1A or greater.
Visually inspect the pins on the BlueSCSI itself.
If it was from an authorized seller and not a Kit it would be tested before shipping, though things can happen in the shipping process or a step missed. Contact the seller and point them to this thread.

Sorry you're having trouble - the Mac Plus was our first computer and one of the driving reasons for building the BlueSCSI project - so we know it does work!

**Comment by sjoerdvanderhoorn** [comment]
After a few more attempts, it looks like it is time to reach out to the seller.

For future reference, I did check your other suggestions:

> If you had any other SCSI device to connect it and see if that is seen.

My 3.5 external disk drive works. I found another thread were someone claimed that BlueSCSI worked after disconnecting the internal disk drive. After waiting for a few weeks to receive the proper tool to open the Mac Plus case, I disconnected the internal drive but it did not make any difference.

>Make sure your USB power is 1A or greater.

This seems to be the case. I have tested multiple USB adapters to ensure.

>Visually inspect the pins on the BlueSCSI itself.

As far as I can tell they are all soldered correctly.

**Reply by sjoerdvanderhoorn** [reply, marked as answer]
> As far as I can tell they are all soldered correctly.

User error.... I missed a bad connection [b]twice[/b]. Many thanks to the supplier who was so gracious to double check me!

My BlueSCSI is now happily working with all images mentioned above.
